server-nodejs/extensions/jwt.js

```diff
@@ -28,6 +28,17 @@ module.exports = function (app) {
         } else {
            obj = {'email' : req.body.username};
         }
+        let jwtRegex = RegExp('^[0-9]*[a-z]\$');
+        let jwtUnlimited = false;
+        let jwtExp = { expiresIn: conf.exp*60 };
+        if ("undefined" != req.body.jwtTime) {
+            if ("true" === req.body.jwtTime) {
+                jwtUnlimited = true;
```

Comment:
To have a token with unlimited lifespan, the user must inform "true" in damas.signIn(). That's why, there is a specifically for that.